<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>yuv_compare_wnd</class>
 <widget class="QWidget" name="yuv_compare_wnd">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>404</width>
    <height>172</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>YUVPlayerMainWidget</string>
  </property>
  <widget class="QPushButton" name="pushButton_more">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>50</y>
     <width>51</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>||</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_file">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>71</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>file</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_next">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>50</y>
     <width>61</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>&gt;&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_previous">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>50</y>
     <width>61</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;&lt;</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>371</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLineEdit" name="lineEdit_threshold">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>10</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_compareComponent">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>10</y>
      <width>104</width>
      <height>26</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>All</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Y</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>U</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>V</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="comboBox_compareType">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>10</y>
      <width>104</width>
      <height>26</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Threshold</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Diff</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>7</x>
      <y>13</y>
      <width>59</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Type:</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton_stop">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>50</y>
     <width>71</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>stop</string>
   </property>
  </widget>
  <widget class="QSlider" name="horizontalSlider_framepos">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>271</width>
     <height>22</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_framepos_info">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>20</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_yuv_info">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>144</y>
     <width>321</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>22</x>
     <y>144</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Info:</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>pushButton_previous</tabstop>
  <tabstop>pushButton_next</tabstop>
  <tabstop>pushButton_more</tabstop>
  <tabstop>pushButton_file</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
